src/PrestaShopBundle/Controller/Admin/Improve/Design/ThemeCatalogController.php

```diff
@@ -26,6 +26,7 @@
 
 namespace PrestaShopBundle\Controller\Admin\Improve\Design;
 
+use PrestaShop\PrestaShop\Core\Foundation\PrestaShopVersion\PrestaShopVersion;
```

Comment:
It's maybe redundant:
`PrestaShop\PrestaShop\Core\Foundation\Version`

Reply:
@PierreRambaud It's because auf PSR 4 autoloading. If there is a directory Version with a class Version this is necessary. Or do I get something wrong?

Reply:
I didn't find anything about PS4 autoloader and version directory :/ https://www.php-fig.org/psr/psr-4/ . Even Symfony doesn't have `Version\Version`, can you show me where do you see that? I think you can remove `Version` directory and have `Version.php` inside `Foundation`, wdyt master @mickaelandrieu ?